ophyd/controls/positioner.py

```diff
@@ -640,6 +655,15 @@ def report(self):
         return {self._name: self.position, 'pv': self._readback.pvname}
 
     @property
+    def describe(self):
+        desc = self._readback.describe[self._readback.name]
+        return {self.name: desc}
+
+    def read(self):
```

Comment:
ATM read() is a method not a property. Open for thoughts?
On Feb 26, 2015, at 4:09 PM, Thomas A Caswell <notifications@github.com<mailto:notifications@github.com>> wrote:


In ophyd/controls/positioner.py<https://github.com/NSLS-II/ophyd/pull/71#discussion_r25464464>:

> @@ -640,6 +655,15 @@ def report(self):
>          return {self._name: self.position, 'pv': self._readback.pvname}
>
>      @property
> +    def describe(self):
> +        desc = self._readback.describe[self._readback.name]
> +        return {self.name: desc}
> +
> +    def read(self):


was this meant to be a property?

—
Reply to this email directly or view it on GitHub<https://github.com/NSLS-II/ophyd/pull/71/files#r25464464>.